### PR TITLE
Add "How regro-cf-autotick-bot create version update PR?" section.

### LINKED ISSUE #1117: DOC: New maintainer
Suggestions for improving the docs in regards to new maintainers and the bot:

- Contributing packages: 
https://conda-forge.org/docs/maintainer/adding_pkgs.html
- Maintaining packages: https://conda-forge.org/docs/maintainer/updating_pkgs.html

  - "Pushing to regro-cf-autotick-bot branch" is maybe a bit obscure to new maintainers.
https://conda-forge.org/docs/maintainer/updating_pkgs.html#pushing-to-regro-cf-autotick-bot-branch

    - [ ] additional / different heading?
    - [ ] how does it know to create a version update / PR?
    - [ ] link to https://github.com/regro/cf-scripts ( cf-regro-autotick-bot )
      - https://regro.github.io/cf-scripts/

### PR DESCRIPTION
**Pull request by ssurbhi560**
Closes #1117 

PR Checklist:

- [x] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] put any other relevant information below

### CODE REVIEW

File: src/maintainer/updating_pkgs.rst [hunk 1]
```diff
@@ -63,6 +63,15 @@ When a new version of a package is released on PyPI/CRAN/.., we have a bot that
    - Checkout the correct branch with remote: ``hub pr checkout 12`` where ``12`` is the ID of the PR.
    - Commit and push on this branch, the remote is automatically set up to push to regro-cf-autotick-bot's fork.
 
+**How does regro-cf-autotick-bot create automatic version updates?**
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+The `regro-cf-autotick-bot <https://github.com/regro/autotick-bot>`__ continuously searches on loop for any PyPI releases, GitHub releases, and any other sources of versions when any updates are released. The source code that gets executed in the loop comes from `cf-scripts <https://github.com/regro/cf-scripts>`__, which contains the code to detect versions and submit PRs, you can read more about cf-scripts `here <https://regro.github.io/cf-scripts/index.html>`__.
```
Comment (jaimergp) [inline src/maintainer/updating_pkgs.rst:68]:
```suggestion
The `regro-cf-autotick-bot <https://github.com/regro/autotick-bot>`__ continuously searches on loop for any PyPI releases, GitHub releases, and any other sources of versions when any updates are released. The source code that gets executed in the loop comes from `cf-scripts <https://github.com/regro/cf-scripts>`__, which contains the code to detect versions and submit PRs. You can read more about ``cf-scripts`` `here <https://regro.github.io/cf-scripts/index.html>`__.
```

**Comment by jaimergp** [comment]
Thanks!

Added some small comments, nothing major. 

The only thing I would change (and probably we have to do this all around the conda-forge docs) is to [avoid using `click here` as the hyperlink text](https://webaccess.berkeley.edu/ask-pecan/click-here).

Other than that, LGTM!

File: src/maintainer/updating_pkgs.rst [hunk 2]
```diff
@@ -65,10 +65,10 @@ When a new version of a package is released on PyPI/CRAN/.., we have a bot that
 
 **How does regro-cf-autotick-bot create automatic version updates?**
 """"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
-The `regro-cf-autotick-bot <https://github.com/regro/autotick-bot>`__ continuously searches on loop for any PyPI releases, GitHub releases, and any other sources of versions when any updates are released. The source code that gets executed in the loop comes from `cf-scripts <https://github.com/regro/cf-scripts>`__, which contains the code to detect versions and submit PRs, you can read more about cf-scripts `here <https://regro.github.io/cf-scripts/index.html>`__.
+The `regro-cf-autotick-bot <https://github.com/regro/autotick-bot>`__ continuously searches on a loop for any PyPI releases, GitHub releases, and any other sources of versions when any updates are released. The source code that gets executed in the loop comes from `cf-scripts repository <https://github.com/regro/cf-scripts>`__, which contains the code to detect versions and submit PRs. Visit `cf-scripts <https://regro.github.io/cf-scripts/index.html>`__ to read more about it.
```
Comment (kathatherine) [inline src/maintainer/updating_pkgs.rst:68]:
```suggestion
The `regro-cf-autotick-bot <https://github.com/regro/autotick-bot>`__ continuously searches on a loop for any PyPI releases, GitHub releases, and any other sources of versions when any updates are released. The source code that gets executed in the loop comes from the `cf-scripts repository <https://github.com/regro/cf-scripts>`__, which contains the code to detect versions and submit PRs. Visit `cf-scripts <https://regro.github.io/cf-scripts/index.html>`__ to read more about it.
```

**Review by kathatherine — APPROVED** [review]
Just one change! All else looks good to me

**Review by kathatherine — APPROVED** [review]
LGTM! @jaimergp, you good to merge this?

**Comment by jaimergp** [comment]
All good! Merging.

**Pull request merged**